action/action.go

```diff
@@ -5,28 +5,12 @@ import (
 
 	"github.com/helm/helm/config"
 	"github.com/helm/helm/log"
+	helm "github.com/helm/helm/util"
```

Comment:
I'm not a fan of generic `util` packages, but that could be just me.  I'm working though it.